ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -514,64 +467,151 @@ describe('AdminInstitutionEditSpec - add mode', () => {
+    await simulateComponentChange(wrapper, findCTEnabled(wrapper), false);
+    expect(findCTEnabled(wrapper).props.checked).toBeFalsy();
+    expect(findCTDetails(wrapper).exists()).toBeFalsy();
   });
 
   it('should update institution tier requirement', async() => {
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
     expect(wrapper).toBeTruthy();
 
-    expect(wrapper.find('[data-test-id="registered-email-domain-input"]').exists()).toBeFalsy();
-    expect(wrapper.find('[data-test-id="registered-email-address-input"]').exists()).toBeFalsy();
-    expect(wrapper.find('[data-test-id="controlled-email-address-input"]').exists()).toBeFalsy();
-    expect(wrapper.find('[data-test-id="controlled-email-domain-input"]').exists()).toBeFalsy();
+    expect(findRTDomainInput(wrapper).exists()).toBeFalsy();
+    expect(findRTAddressInput(wrapper).exists()).toBeFalsy();
+    expect(findCTAddressInput(wrapper).exists()).toBeFalsy();
+    expect(findCTDomainInput(wrapper).exists()).toBeFalsy();
 
-    const agreementTypeDropDown = wrapper.find('[data-test-id="registered-agreement-dropdown"]').instance() as Dropdown;
-    await simulateComponentChange(wrapper, agreementTypeDropDown, InstitutionMembershipRequirement.ADDRESSES);
+    await simulateComponentChange(wrapper, findRTDropdown(wrapper), InstitutionMembershipRequirement.ADDRESSES);
 
-    expect(wrapper.find('[data-test-id="registered-email-address-input"]').exists()).toBeTruthy();
-    expect(wrapper.find('[data-test-id="registered-email-address-input"]').first().prop('value')).toBe('');
+    expect(findRTAddressInput(wrapper).exists()).toBeTruthy();
+    expect(findRTAddressInput(wrapper).first().prop('value')).toBe('');
 
-    expect(wrapper.find('[data-test-id="registered-email-domain-input"]').exists()).toBeFalsy();
-    expect(wrapper.find('[data-test-id="controlled-email-address-input"]').exists()).toBeFalsy();
-    expect(wrapper.find('[data-test-id="controlled-email-domain-input"]').exists()).toBeFalsy();
+    expect(findRTDomainInput(wrapper).exists()).toBeFalsy();
+    expect(findCTAddressInput(wrapper).exists()).toBeFalsy();
+    expect(findCTDomainInput(wrapper).exists()).toBeFalsy();
 
-    wrapper.find('[data-test-id="registered-email-address-input"]').first()
-        .simulate('change', {target: {value: 'user@domain.com'}});
-    wrapper.find('[data-test-id="registered-email-address-input"]').first()
-        .simulate('blur');
+    findRTAddressInput(wrapper).first().simulate('change', {target: {value: 'user@domain.com'}});
+    findRTAddressInput(wrapper).first().simulate('blur');
 
     // RT no change
-    expect(wrapper.find('[data-test-id="registered-email-address-input"]').first().prop('value'))
-        .toBe('user@domain.com');
+    expect(findRTAddressInput(wrapper).first().prop('value')).toBe('user@domain.com');
+  });
+
+  it('should not change eRA Required and tier enabled switches when changing tier requirement', async() => {
```

Comment:
these new Add Mode tests were copied from Edit Mode and lightly edited, passing without app-code changes.